components/speedreader/speedreader_whitelist.cc

```diff
@@ -8,29 +8,54 @@
 #include <utility>
```

Comment:
for followup the name/class of this file should really be SpeedreaderWhitelistInstaller